images.yaml

```diff
@@ -90,24 +75,202 @@ aliases:
   valohai/keras:2.0.0-theano0.8.2-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04: 'Keras 2.0.0 with Theano 0.8.2 backend and GPU support on Python 3'
   valohai/keras:2.0.0-theano0.9.0rc4-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04: 'Keras 2.0.0 with Theano 0.9.0rc4 backend and GPU support on Python 3'
+
+details:
+  tensorflow/tensorflow:1.5.0-devel-gpu-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.5.0 on Python 3 with GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.5.0'
+      - 'GPU'
+  tensorflow/tensorflow:1.9.0-devel-gpu-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.9.0 on Python 3 with GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.9.0'
+      - 'GPU'
+  tensorflow/tensorflow:1.9.0-devel-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.9.0 developer version on Python 3 without GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.5.0'
+      - 'CPU'
+  tensorflow/tensorflow:1.13.1-gpu-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.13.1 on Python 3 with GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.13.1'
+      - 'GPU'
+  tensorflow/tensorflow:1.13.1-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.13.1 on Python 3 without GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.13.1'
+      - 'CPU'
+  tensorflow/tensorflow:2.0.0-gpu-py3:
+    isRecommended: True
+    description: 'TensorFlow 2.0.0 on Python 3 with GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 2.2.0'
+      - 'GPU'
+  tensorflow/tensorflow:1.15.2-gpu-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.15.2 on Python 3 with GPU support'
+    tags:
+      - 'python 2'
+      - 'tensorflow 1.15.2'
+      - 'GPU'
+  tensorflow/tensorflow:1.15.0-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.15.0 on Python 3 without GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.15.0'
+      - 'CPU'
+  tensorflow/tensorflow:1.15.0-gpu-py3:
+    isRecommended: False
+    description: 'TensorFlow 1.15.0 on Python 3 with GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 1.15.0'
+      - 'GPU'
+  tensorflow/tensorflow:2.0.1-gpu-py3:
+    isRecommended: True
+    description: 'TensorFlow 2.0.1 on Python 3 with GPU support'
+    tags:
+      - 'python 3'
+      - 'tensorflow 2.0.1'
+      - 'GPU'
+  ufoym/deepo:all-py36:
+    isRecommended: True
+    description: 'All common deep learning frameworks on Python 3 with GPU support)'
+    tags:
+      - 'python 3.6'
+      - 'GPU'
+      - 'CUDA 10.0'
+      - 'cuDNN 7'
+      - 'tensorflow'
+      - 'sonnet'
+      - 'torch 7'
+      - 'keras'
+      - 'mxnet'
+      - 'cntk'
+      - 'chainer'
+      - 'theano'
+      - 'lasagne'
+      - 'caffe 1.0.0'
+      - 'caffe2'
+  ufoym/deepo:all-py36-cpu:
```

Comment:
I hope I got the tags right on this, but there were no versions stated and not sure if cuDNN can be on a non-gpu variant?

Reply:
Not sure how this should be read... => https://github.com/ufoym/deepo#Comparison

Reply:
cuDNN is the CUDA deep-learning library, so anything with it is GPU-flavored.